diff --git a/node_modules/chalk/license b/node_modules/chalk/license
old mode 100644
new mode 100755
diff --git a/node_modules/chalk/readme.md b/node_modules/chalk/readme.md
old mode 100644
new mode 100755
diff --git a/node_modules/chalk/source/index.d.ts b/node_modules/chalk/source/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/chalk/source/index.js b/node_modules/chalk/source/index.js
old mode 100644
new mode 100755
index 302024b..aeee231
--- a/node_modules/chalk/source/index.js
+++ b/node_modules/chalk/source/index.js
@@ -1,5 +1,5 @@
-import ansiStyles from '#ansi-styles';
-import supportsColor from '#supports-color';
+import ansiStyles from './vendor/ansi-styles';
+import supportsColor from './vendor/supports-color';
 import { // eslint-disable-line import/order
 	stringReplaceAll,
 	stringEncaseCRLFWithFirstIndex,
diff --git a/node_modules/chalk/source/utilities.js b/node_modules/chalk/source/utilities.js
old mode 100644
new mode 100755
diff --git a/node_modules/chalk/source/vendor/ansi-styles/index.d.ts b/node_modules/chalk/source/vendor/ansi-styles/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/chalk/source/vendor/ansi-styles/index.js b/node_modules/chalk/source/vendor/ansi-styles/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/chalk/source/vendor/supports-color/browser.d.ts b/node_modules/chalk/source/vendor/supports-color/browser.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/chalk/source/vendor/supports-color/browser.js b/node_modules/chalk/source/vendor/supports-color/browser.js
old mode 100644
new mode 100755
diff --git a/node_modules/chalk/source/vendor/supports-color/index.d.ts b/node_modules/chalk/source/vendor/supports-color/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/chalk/source/vendor/supports-color/index.js b/node_modules/chalk/source/vendor/supports-color/index.js
old mode 100644
new mode 100755
